tests/h/search/index_test.py

```diff
@@ -310,6 +312,109 @@ def test_annotation_is_marked_deleted(self, es_client, factories, index, search)
         assert result.get('_source').get('deleted') is True
 
 
+class TestBatchIndexer(object):
+    def test_it_indexes_all_annotations(self, batch_indexer, es_client, factories):
+        annotations = factories.Annotation.create_batch(3)
+        ids = [a.id for a in annotations]
+
+        batch_indexer.index()
+
+        for id in ids:
+            result = es_client.conn.get(index=es_client.index,
+                                        doc_type="annotation",
+                                        id=id)
+            assert result["_id"] == id
+
+    def test_it_indexes_specific_annotations(self, batch_indexer, es_client, factories):
+        annotations = factories.Annotation.create_batch(5)
+        ids = [a.id for a in annotations]
+        ids_to_index = ids[:3]
+        ids_not_to_index = ids[3:]
+
+        batch_indexer.index(ids_to_index)
+
+        for id in ids_to_index:
+            result = es_client.conn.get(index=es_client.index,
+                                        doc_type="annotation",
+                                        id=id)
+            assert result["_id"] == id
+
+        for id in ids_not_to_index:
```

Comment:
For these tests we could grab all the annotations instead of looping through each of them individually; it looks like if you leave the search body empty it will grab all of them:
``` 
       res = conn.search()['hits'['hits']
       found_ids = [ann._id for ann in res]
       assert found_ids.sort() == ids_to_index.sort()
```
I think this would make the tests a lot simpler as you don't have to check for non indexed and indexed annotations.